modules/modify-rhsso-requests-limits.adoc

```diff
@@ -0,0 +1,30 @@
+// Module is included in the following assemblies:
+//
+// * installing-red-hat-openshift-gitops
+
+:_content-type: PROCEDURE
+[id="modifying-rhsso-resource-requests-limits_{context}"]
+= Modifying RHSSO resource requests/limits
+
+[role="_abstract"]
+By default, the RHSSO container is created with default resource requests and limits as shown below:
+
+|===
+|*Resource* |*Requests* |*Limits*
+
+|CPU|500|1000m
+|Memory|512 Mi|1024 Mi
+
+|===
+
+You can modify the default resource requirements patching the Argo CD CR:
```

Comment:
Make it a procedure step:

- Modify the default resource requirements patching the Argo CD CR: